tools/actions/commit-queue.sh

```diff
@@ -64,7 +64,8 @@ for pr in "$@"; do
   if ! tail -n 10 output | grep '. Post "Landed in .*/pull/'"${pr}"; then
     gitHubCurl "$(labelsUrl "$pr")" POST --data '{"labels": ["'"${COMMIT_QUEUE_FAILED_LABEL}"'"]}'
 
-    jq -n --arg content "<details><summary>Commit Queue failed</summary><pre>$(cat output)</pre></details>" '{body: $content}' > output.json
+    cqurl="Commit Queue action: $GITHUB_SERVER_URL/${OWNER}/${REPOSITORY}/actions/runs/$GITHUB_RUN_ID"
```

Comment:
This variable could be set outside the for loop, but there's no harm in having it here so 🤷‍♀️

Reply:
I agree, but since this is bash and everything is kind of 'messy' (imo) I'd prefer to have it right beside the place of usage if you don't mind.